src/formulas.js

```diff
@@ -888,4 +889,19 @@ function ceiling(number, significance) {
     return Math.ceil(number / significance) * significance
 }
 
+function filter(range, conditions) {
```

Comment:
I'm debating whether we need to have this somewhat limited FILTER function here. If you have a look at this PR here https://github.com/formulajs/formulajs/pull/119 I've created a more in-depth, tested FILTER function. Feel free to add the full logic of it here as well though (if you like).

Reply:
while formulajs guys don't merge I put your logic here on our side